enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/TopicImpl.java

```diff
@@ -200,4 +225,49 @@ public void setModelId(String modelId) {
 		
 	}
 
+	@Override
+	public String getTopicURI() throws Exception {
+		if (this.topicID == 0L)
+		{
+			throw new Exception("Topic ID is undefined");
+			
+		}
+		
+		return UriUtils.buildEntityIdUri(this.baseURI,this.type,String.valueOf(this.topicID));
+	}
+
+	@Override
+	public void setBaseURI(String baseURI) {
+		this.baseURI = baseURI;
+		
+	}
+
+	@Override
+	public String getBaseURI() {
+		return this.baseURI;
+	}
+
+	@Override
+	public String getType() {
+		return this.type;
+	}
+	
+	public TopicToText createTopicToText()
```

Comment:
This method is not needed, the serialization is done directly with the jackson annotations.